news/4305.hooks.rst

```diff
@@ -0,0 +1,8 @@
+Add hook for pyexcel
+Add hook for pyexcel-io
+Add hook for pyexcel-ods
+Add hook for pyexcel-ods3
+Add hook for pyexcel-odsr
+Add hook for pyexcel-xls
+Add hook for pyexcel-xlsx
+Add hook for pyexcel-xlsxw
```

Comment:
Thanks for being so precise. OTOH for the readers of the release-notes "Added hook for pyexcel" would be enough information, What dou you think?

Reply:
Or "Added hook for pyexcel and its plugins odsr, ...)

Reply:
You're right. I will change it.